### DISCUSSION TITLE
Cloning repository

### DISCUSSION THREAD

**Discussion by themystic01** [community/community#36017] [Repositories] [answered]
Is there a way to get a repository's link for cloning?

**Comment by airtower-luna** [comment, marked as answer]
Go to the repository page, click the green "Code" button, and select the protocol you want. Copy the URL shown in the popup, that's all. :slightly_smiling_face:

**Reply by themystic01** [reply]
Thank you for your help.

**Comment by Anmol-Baranwal** [comment]
You can see the popup window below: 
![Cloning Repository](https://user-images.githubusercontent.com/74038190/195603228-a08a218b-2c99-4032-8012-3fe6424c7a5b.png)

If you're new to _git_ and/or _GitHub_, I suggest you go through [the GitHub Guides](https://guides.github.com/introduction/flow/).

Using the below steps, you can clone the repository by using the information on the new window- 

1. Fork the repository
2. Clone the fork via `git`

   - Using SSH

     ```shell
     git clone --filter=tree:0 git@github.com:Anmol-Baranwal/Awesome-Illustrations-4Projects.git
     ```

   - Using HTTPS

     ```shell
     git clone --filter=tree:0 https://github.com/Anmol-Baranwal/Awesome-Illustrations-4Projects.git
     ```

   - Using GitHub CLI

     ```shell
     gh repo clone Anmol-Baranwal/Awesome-Illustrations-4Projects -- --filter=tree:0
     ```

**Reply by themystic01** [reply]
I already did that. Thanks for your help.
